pkg/build/controller/config_controller.go

```diff
@@ -55,9 +55,15 @@ func (c *BuildConfigController) HandleBuildConfig(bc *buildapi.BuildConfig) erro
 	}
 
 	glog.V(4).Infof("Running build for BuildConfig %s/%s", bc.Namespace, bc.Name)
+
+	buildTriggerCauses := []buildapi.BuildTriggerCause{}
 	// instantiate new build
 	lastVersion := 0
 	request := &buildapi.BuildRequest{
+		TriggeredBy: append(buildTriggerCauses,
+			buildapi.BuildTriggerCause{
+				Reason: "Triggered by config change",
```

Comment:
maybe, but if we change it we need to remove "triggered" from all the other reason codes as well, since it is redundant there too.